bot.user.js

```diff
@@ -465,14 +461,103 @@ window.collisionCheck = function(circle1, circle2) {
         return false;
     }
 };
-
+// Sort collision points based on distance
+window.getCollisionPoints = function() {
```

Comment:
Sorry, didn't see that the other collision points was used in the 'goodPath' function.